### DISCUSSION TITLE
addcart problem

### DISCUSSION THREAD

**Discussion by eruruu** [chr233/ASFEnhance#103] [提问求助 / Support]
I tried to use the addcart command but can't make it happen, shopping cart keep showing empty, what's wrong with my input?

this is the command output

[doge@asf script]$ curl -X POST "http://192.168.2.6:1212/Api/Command" -H "accept: application/json" -H "Content-Type: application/json" -d "{\"Command\":\"asfenhance\"}"
{"Result":"<ASFE> ASFEnhance Version 1.7.25.0","Message":"OK","Success":true}

[doge@asf script]$ curl -X POST "http://192.168.2.6:1212/Api/Command" -H "accept: application/json" -H "Content-Type: application/json" -d "{\"Command\":\"cart k04\"}"
{"Result":"<k04> Shopping cart is empty\n","Message":"OK","Success":true}

[doge@asf script]$ curl -X POST "http://192.168.2.6:1212/Api/Command" -H "accept: application/json" -H "Content-Type: application/json" -d "{\"Command\":\"addcart k04 322330\"}"
{"Result":"<k04> ID: 322330 | Status: OK\n","Message":"OK","Success":true}

[doge@asf script]$ curl -X POST "http://192.168.2.6:1212/Api/Command" -H "accept: application/json" -H "Content-Type: application/json" -d "{\"Command\":\"cart k04\"}"
{"Result":"<k04> Shopping cart is empty\n","Message":"OK","Success":true}
[doge@asf script]$

**Comment by chr233** [comment]
🤔
